tests/integ/modin/groupby/test_groupby_negative.py

```diff
@@ -559,20 +560,12 @@ def test_groupby_agg_invalid_min_count(
 
 
 @sql_count_checker(query_count=0)
-def test_groupby_var_no_support_for_timedelta():
-    native_df = native_pd.DataFrame(
-        {
-            "A": native_pd.to_timedelta(
-                ["1 days 06:05:01.00003", "15.5us", "nan", "16us"]
-            ),
-            "B": [8, 8, 12, 10],
-        }
-    )
-    snow_df = pd.DataFrame(native_df)
-    with pytest.raises(
-        NotImplementedError,
-        match=re.escape(
-            "SnowflakeQueryCompiler::groupby_agg is not yet implemented for Timedelta Type"
+def test_timedelta_var_invalid():
```

Comment:
why it works now?

Reply:
formerly `var` was missing from the maps [here](https://github.com/snowflakedb/snowpark-python/blob/fe51d4d167760bb0a858de0e404e9d7a644fb3a9/src/snowflake/snowpark/modin/plugin/_internal/aggregation_utils.py#L275-L289), so we would raise a `NotImplementedError` [in the query compiler](https://github.com/snowflakedb/snowpark-python/blob/fe51d4d167760bb0a858de0e404e9d7a644fb3a9/src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py#L3592).